GwApiCore/src/main/java/us/gameandwatching/gwapi/core/MatchResult.java

```diff
@@ -0,0 +1,44 @@
+package us.gameandwatching.gwapi.core;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+import com.hubspot.rosetta.annotations.RosettaCreator;
+import com.hubspot.rosetta.annotations.RosettaValue;
+
+public enum MatchResult {
+  PLAYER_1_WON(1, true, false),
+  PLAYER_2_WON(2, false, true),
+  DRAW(3, false, false);
+
+  private static Map<Integer, MatchResult> MATCH_RESULTS_BY_ID = Maps.uniqueIndex(Arrays.asList(values()), MatchResult::getId);
+
+  private final int id;
+  private final boolean player1Won;
+  private final boolean user2won;
+
+  MatchResult(int id, boolean user1won, boolean user2won) {
+    this.id = id;
+    this.player1Won = user1won;
```

Comment:
Possibly. As you can probably tell I got halfway through implementing it and changed my mind since I liked player better